internal/colorize/wrap.go

```diff
@@ -0,0 +1,161 @@
+package colorize
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+	"unicode/utf8"
+)
+
+type WrappedLines []WrappedLine
+
+type WrappedLine struct {
+	Line   string
+	Length int
+}
+
+func (c WrappedLines) String() string {
+	var result string
+	for _, wrapped := range c {
+		result = result + wrapped.Line
+	}
+
+	return result
+}
+
+var indentRegexp = regexp.MustCompile(`^([ ]+)`)
+var isLinkRegexp = regexp.MustCompile(`\s*(\[[^\]]+\])?https?://`)
+
+func Wrap(text string, maxLen int, includeLineEnds bool, continuation string) WrappedLines {
```

Comment:
Despite me doing a rename, GitHub thinks this entire file is new. I refactored this method and added `colorTags()` below. That's it. Everything else is the same.